feder/institutions/viewsets.py

```diff
@@ -19,15 +23,61 @@ class Meta:
         fields = ["name", "tags", "jst", "regon"]
 
 
+class InstitutionCSVRenderer(PaginatedCSVRenderer):
+    header = [
+        "pk",
+        "name",
+        "email",
+        "jst",
+        "jst_name",
+        "jst_voivodeship",
+        "jst_category",
+        "created",
+        "modified",
+        "regon",
+        "self",
+        "slug",
+        "tags.0",
```

Comment:
Why `.0`?

Reply:
This is how this CSV renderer displays m2m relationships. By creating separate column for each related object. It's not crucial for this task, so I didn't want to put much effort into that, just overridden the default label.

Reply:
In my opinion, the correct export of tags is also important to properly manage institutions. Maybe just separate it with commas or a semicolon to return everything?